pkg/storegateway/bucket_stores.go

```diff
@@ -310,7 +311,7 @@ func (u *BucketStores) syncUsersBlocks(ctx context.Context, f func(context.Conte
 }
 
 // Series implements the storepb.StoreServer interface, making a series request to the underlying user bucket store.
-func (u *BucketStores) Series(req *storepb.SeriesRequest, srv storepb.Store_SeriesServer) error {
+func (u *BucketStores) Series(req *storepb.SeriesRequest, srv storegatewaypb.StoreGateway_SeriesServer) error {
```

Comment:
Note to reviewers: this is **not** a breaking change because `storepb.Store_SeriesServer` and `storegatewaypb.StoreGateway_SeriesServer` interfaces are **identical**.